modules/deployments-comparing-deploymentconfigs.adoc

```diff
@@ -18,6 +18,29 @@ For `DeploymentConfig` objects, if a node running a deployer pod goes down, it w
 
 However, deployment rollouts are driven from a controller manager. The controller manager runs in high availability mode on masters and uses leader election algorithms to value availability over consistency. During a failure it is possible for other masters to act on the same deployment at the same time, but this issue will be reconciled shortly after the failure occurs.
 
+[id="delpoyments-specific-features_{context}"]
+== Deployment-specific features
+
+[discrete]
+==== Rollover
+
+The deployment process for `Deployment` objects is driven by a controller loop, in contrast to `DeploymentConfig` objects which use deployer pods for every new rollout. This means that the `Deployment` object can have as many active replica sets as possible, and eventually the deployment controller will scale down all old replica sets and scale up the newest one.
+
+`DeploymentConfig` objects can have at most one deployer pod running, otherwise multiple deployers end up conflicting while trying to scale up what they think should be the newest replication controller. Because of this, only two replication controllers can be active at any point in time. Ultimately, this translates to faster rapid rollouts for `Deployment` objects.
```

Comment:
Agreed that this sentence needs some work. I cut out some of it. @atiratree Can you let me know if this retains an accurate meaning:

```suggestion
`DeploymentConfig` objects can have at most one deployer pod running, otherwise multiple deployers might conflict when trying to scale up the newest replication controller. Because of this, only two replication controllers can be active at any point in time. Ultimately, this translates to faster rapid rollouts for `Deployment` objects.
```